src/support/rgw_support_bundle

```diff
@@ -39,32 +42,33 @@ class SupportBundleError(BaseError):
 
 class RGWSupportBundle:
     """Generate rgw specific support bundle."""
-    _ceph_conf_file = '/etc/ceph/ceph.conf'
-    _ceph_crashdump_dir = '/var/log/ceph/crash'
     _tmp_src = '/tmp/rgw/'
 
     @staticmethod
     def generate(bundle_id: str, target_path: str, cluster_conf: str, **filters):
         """ Generate a tar file. """
         coredumps = filters.get('coredumps', False)
-        cortx_config_store = MappedConf(cluster_conf)
-        log_base = cortx_config_store.get('cortx>common>storage>log')
         machine_id = Conf.machine_id
+        cortx_config_store = MappedConf(cluster_conf)
+        log_base = cortx_config_store.get(LOG_PATH_KEY)
+        config_path = cortx_config_store.get(CONFIG_PATH_KEY)
+        rgw_config_dir = os.path.join(config_path, COMPONENT_NAME, machine_id)
+        rgw_config_file = os.path.join(rgw_config_dir, RGW_CONF_FILE)
```

Comment:
Can we remove rgw from these variable names?

Reply:
yes done.
removed 'rgw' reference from var names.
But the classname & filename still rgw reference.